spec/support/script_tag_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ScriptTagHelper
```

Comment:
This name feels too generic to me... but I couldn't think of anything more meaningful... 🤷‍♀️

Reply:
`JsSdkScriptHelper`? 
…but I agree, it doesn't matter much

Reply:
`PaypalSdkScriptTagHelper` maybe?